code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1110,6 +1110,34 @@ Traitors and the like can also be revived with the previous role mostly intact.
 			continue
 		A.current.AbnoRadio()
 
+// Selects and starts a core suppression
+/client/proc/InitCoreSuppression()
+	set category = "Admin.Fun"
+	set name = "Start Core Suppression"
+	if(!check_rights(R_ADMIN) || !check_rights(R_FUN))
+		return
+
+	if(istype(SSlobotomy_corp.core_suppression))
+		var/confirm = alert(src, "Another Core Suppression is already in progress. Are you sure you want to proceed?", "Replace suppression", "Yes", "No")
+		if(confirm != "Yes")
+			return
+	var/datum/suppression/core_type = input("Select core suppression type","Set Core Type") as null|anything in subtypesof(/datum/suppression)
+	if(!ispath(core_type))
+		return
+	var/run_white = alert(src, "Do you wish to queue white ordeals?", "White Ordeals", "Yes", "No", "Cancel")
+	var/white_ordeals = TRUE
+	if(!(run_white in list("Yes", "No")))
+		return
+	if(run_white == "No")
+		white_ordeals = FALSE
+	SSlobotomy_corp.core_suppression = new core_type
```

Comment:
Even if it's admemes, if something like Training Suppression is replaced by this, employees would be stuck with a -40 to all stats, no? The Ordeal `End` proc should probably contain a bool parameter that when set to `True` simply ends it without providing rewards, so it can be properly replaced.